Fenced_Frames_Ads_Reporting.md

```diff
@@ -159,8 +166,27 @@ Currently, the only `eventType` that `setReportEventDataForAutomaticBeacons` all
 
 If invoked multiple times, the latest invocation before the top-level navigation would be the one that’s honored.
 
-`eventData` can be empty, in which case the automatic beacon will still be sent but without an event data body in the HTTP request.
+`eventData` is optional, and can be empty. If `eventData` is not specified, or is empty, the automatic beacon will still be sent but without an event data body in the HTTP request.
 
 If `setReportEventDataForAutomaticBeacons` is not invoked, the browser will not send an automatic beacon because the `destination` is unknown.
 
+# Support for Ad Components
+## Goal
+When a rendered ad is composed of [multiple pieces](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#34-ads-composed-of-multiple-pieces), it is useful to detect user clicks that happened on ad components.
+
+## Design
+### Event Type and Reporting Destination
+For fenced frames rendering the ad components under the top-level ad fenced frame, the `reserved.top_navigation` event type and corresponding reporting destination registered for the top-level fenced frame are reused when beacons are sent from the ad component fenced frames.
 
+### Restricted to send `reserved.top_navigation` beacons only
+* Invocation of the `reportEvent` API from an ad component fenced frame is disallowed.
+* The only supported beacon to be sent from an ad component fenced frame is the `reserved.top_navigation` automatic beacon. Note this beacon is gated on a user activation. 
```

Comment:
Can we be explicit about what "a user activation" is?  is this a click?

Reply:
Changed to mention click.